docs/topics/dokka-migration.md

```diff
@@ -209,12 +209,12 @@ options according to your project setup:
     customAssets.from("example.png", "example2.png")
     ```
 
-* **Output directory:** Use the `dokka {}` block to specify a single output directory for all Dokka-generated documentation.
+* **Output directory:** Use the `dokka {}` block to specify the output directory for generated Dokka documentation.
 
     Previous configuration:
 
     ```kotlin
-    tasks.dokkaHtml{
+    tasks.dokkaHtml {
         dokkaSourceSets {
```

Comment:
BTW: `previous configuration` is wrong, as `outputDirectory` should be at top-level of `dokkaHtml` configuration.
Could you please adapt it?

Reply:
Like this?

````
```kotlin
tasks.dokkaHtml {
    outputDirectory.set(layout.buildDirectory.dir("dokkaDir"))
}
```
````

Reply:
Yeah, thanks!